pyproject.toml

```diff
@@ -68,20 +68,20 @@ coloredlogs = ">=10,<16"
 sanic = "^21.12.0"
 typing-extensions = ">=4.1.1,<5.0.0"
 Sanic-Cors = "^2.0.0"
-prompt-toolkit = "^3.0,<3.0.29"
+prompt-toolkit = ">=3,<4"
 "ruamel.yaml" = ">=0.16.5,<0.18.0"
 
 [tool.poetry.dev-dependencies]
-pytest-cov = "^3.0.0"
+pytest-cov = "^4.0.0"
 coveralls = "^3.0.1"
 pytest = "^7.1.3"
-black = "22.8.0"
+black = "22.10.0"
 flake8 = "^5.0.4"
 flake8-docstrings = "^1.5.0"
 questionary = ">=1.5.1,<1.11.0"
 towncrier = "^22.8.0"
 toml = "^0.10.0"
 pep440-version-utils = "^0.3.0"
 semantic_version = "^2.8.5"
-mypy = "^0.971"
+mypy = "^0.991"
```

Comment:
Please double-check if any pre-commit hook needs updating too.

Optional: Would be great if these updates also had the corresponding updates in rasa.